src/modules/zerion-api/hooks/useAssetChart.ts

```diff
@@ -0,0 +1,13 @@
+import { useQuery } from '@tanstack/react-query';
+import { ZerionAPI } from 'src/modules/zerion-api/zerion-api.client';
+import { type Params } from '../requests/asset-get-chart';
+
+export function useAssetChart(params: Params) {
+  return useQuery({
+    queryKey: ['assetGetChart', params],
+    queryFn: () => ZerionAPI.assetGetChart(params),
+    suspense: false,
+    keepPreviousData: true,
+    staleTime: 1000 * 60,
```

Comment:
I wonder if this should be 0?
We are refetching chart data on the web for each period change. I think it's expected that changing chart period acts as a "refresh" of data